app/src/main/java/org/wikipedia/suggestededits/ImageRecsOnboardingFragment.kt

```diff
@@ -64,6 +65,10 @@ class ImageRecsOnboardingFragment : OnboardingFragment(false), OnboardingPageVie
             if (OnboardingPage.PAGE_CONSENT.code() == position) {
                 view.setSwitchChecked(Prefs.isImageRecsConsentEnabled())
             }
+            if (DimenUtil.displayHeightPx < 1200) {
+                // Remove layout gravity of the text below on small screens to make centered image visible
+                view.removeMainTextLayoutGravity()
+            }
```

Comment:
Looks like this logic should be inside the `OnboardingPageView`.

Reply:
We can move this inside after this release @cooltey ... but if i do now, then I will haave to test all onboarding screens to see if they are still working on small screen ... so I can add a todo. Does that sound ok?

Reply:
Sounds good.